src/components/UserManagement/UserTableData.jsx

```diff
@@ -11,6 +11,7 @@ import { boxStyle } from 'styles';
 import { connect } from 'react-redux';
 import { formatDate } from 'utils/formatDate';
 import { cantUpdateDevAdminDetails } from 'utils/permissions';
+import { useSelector } from 'react-redux';
```

Comment:
At the bottom of this file, there's a `connect(null, { hasPermission })...` instead of having to import/use `useSelector` or `useDispatch` for everything, you can use mapStateToProps and mapDispatchToProps. Either way works, but connect() is a bit cleaner (all your redux state/functions in one place) and more in line with the coding practices used in this codebase.
https://react-redux.js.org/api/connect